packages/components/action-button/README.md

```diff
@@ -0,0 +1,131 @@
+<h1 align="center">
+    <a href="https://tradeshift.com/">
+      <img alt="Tradeshift" src="https://tradeshift.com/wp-content/themes/Tradeshift/img/brand/logo-black.png"/>
+    </a>
+</h1>
+
+<h1 align="center">Elements - action-button</h1>
+
+<p align="center">
+  Part of the reusable Tradeshift UI Components as Web Components.
+    <a href="https://tradeshift.github.io/elements/?path=/story/ts-action-button--default">
+      Demo
+    </a>
+</p>
+
+<p align="center">
+    <a href="https://www.npmjs.com/package/@tradeshift/elements.action-button">
+      <img alt="NPM Version" src="https://badgen.net/npm/v/@tradeshift/elements.action-button" height="20"/>
+    </a>
+    <a href="https://npmcharts.com/compare/@tradeshift/elements.action-button?minimal=true">
+		  <img alt="Downloads per month" src="https://badgen.net/npm/dm/@tradeshift/elements.action-button" height="20"/>
+		</a>
+		<a href="https://www.npmjs.com/browse/depended/@tradeshift/elements.action-button">
+		  <img alt="Dependent packages" src="https://badgen.net/npm/dependents/@tradeshift/elements.action-button" height="20"/>
+		</a>
+</p>
+
```

Comment:
Please, add the table with properties and events. You can use the `ts-header` README as an example.

Reply:
Done :ok_hand: